packages/service-library/src/servicelib/aiohttp/tracing.py

```diff
@@ -72,8 +73,25 @@ def setup_tracing(
 
     # Add the span processor to the tracer provider
     tracer_provider.add_span_processor(BatchSpanProcessor(otlp_exporter))  # type: ignore[attr-defined] # https://github.com/open-telemetry/opentelemetry-python/issues/3713
-    # Instrument aiohttp server and client
+    # Instrument aiohttp server
     AioHttpServerInstrumentor().instrument()
+    # Explanation for extra call DK 10/2024:
```

Comment:
Let's create a simple test that continuously verifies all these observations
This test can also evaluate
- how it affects other existing middlewares
- what is the overhead that instrumentation

As an example, I suggest to see how we analyzes our middlewares in this [test-suite](https://github.com/itisfoundation/osparc-simcore/blob/d7bb29e01dbfd0f276b2c6293cb32e05ada1b458/packages/service-library/tests/aiohttp/test_rest_middlewares.py)

Reply:
Just as info. I started noticing this warning in the tests
```
../../.pyenv/versions/3.11.6/lib/python3.11/abc.py:106
  /home/crespo/.pyenv/versions/3.11.6/lib/python3.11/abc.py:106: DeprecationWarning: Inheritance class _InstrumentedApplication from web.Application is discouraged
    cls = super().__new__(mcls, name, bases, namespace, **kwargs)
```
which comes from `opentelemtry`. 
![image](https://github.com/user-attachments/assets/170bdbab-1e17-4926-b4ac-f48207476b6c)

Reply:
This warning should be resolved now due to removing the call to `AioHttpServerInstrumentor().instrument()` entirely